docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  bot:
+    depends_on:
+      - db
+    build: ./
+    env_file: .env
+    command: bash -c "sleep 1.5 && /bot/start.sh"
+
+  db:
+    image: postgres:latest
```

Comment:
maybe you can specify a version. otherwise a new version of Postgres may contain breaking changes

Reply:
[PostgreSQL 14.5 version](https://github.com/pxp9/weather_bot_rust/pull/24/commits/d4280f16bf66d741e9f83d332a36ebdd4e3703c0)